configure.cmake

```diff
@@ -855,6 +855,16 @@ IF(NOT CMAKE_CROSSCOMPILING AND NOT MSVC)
     }
    " HAVE_FAKE_PAUSE_INSTRUCTION)
   ENDIF()
+  IF (NOT HAVE_PAUSE_INSTRUCTION)
+    CHECK_C_SOURCE_COMPILES("
+    int main()
+    {
+     __asm__ __volatile__ (\"or 1,1,1\");
```

Comment:
sets the hardware thread priority to low. 2,2,2 restores it to the default medium. Next commit will use the wrappers to achieve the same thing and will look more readable.